mlir/lib/Conversion/GPUCommon/GPUToLLVMConversion.cpp

```diff
@@ -167,7 +167,8 @@ class ConvertOpToGpuRuntimeCallPattern : public ConvertOpToLLVMPattern<OpTy> {
       "mgpuMemAlloc",
       llvmPointerType /* void * */,
       {llvmIntPtrType /* intptr_t sizeBytes */,
-       llvmPointerType /* void *stream */}};
+       llvmPointerType /* void *stream */,
+       llvmInt64Type /* bool isHostShared */}};
```

Comment:
I think it should be `i8`  to match C++ bool, or, better just use i32 (int) on both sides.